hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/container/balancer/ContainerBalancerConfiguration.java

```diff
@@ -123,6 +123,14 @@
           "This configuration is false by default.")
   private boolean networkTopologyEnable = false;
 
+  @Config(key = "trigger.du.before.move.enable", type = ConfigType.BOOLEAN,
+      defaultValue = "false", tags = {ConfigTag.BALANCER},
+      description = "whether to send command to all the data nodes to run du " +
+          "immediately before starting a balance iteration. note that " +
+          "running du is very time consuming , especially when the disk " +
+          "usage rate of a data node is very high")
```

Comment:
We can update the description to `all healthy, in-service datanodes` or something similar.